data/pgroonga--4.0.0--3.2.5.sql

```diff
@@ -1 +1,23 @@
 -- Downgrade SQL
+
+DO LANGUAGE plpgsql $$
+BEGIN
+	PERFORM setting FROM pg_settings
+	 WHERE name = 'server_version' AND
+	       setting LIKE '%Postgres-XL%';
```

Comment:
Postgres-XL is a dead project. (We can't access to www.postgres-xl.org now.)

So we can remove this condition entirely.